packages/pluggableWidgets/maps-web/src/Maps.tsx

```diff
@@ -14,7 +14,10 @@ export default function Maps(props: MapsContainerProps): ReactNode {
     const [locations] = useLocationResolver(
         props.markers,
         props.dynamicMarkers,
-        props.geodecodeApiKeyExp?.value ?? props.geodecodeApiKey ?? props.apiKeyExp?.value ?? props.apiKey
+        props.geodecodeApiKeyExp?.value ??
+            (props.geodecodeApiKey || undefined) ??
```

Comment:
Why is this `|| undefined` construction necessary?

Reply:
Due to empty string being passed as "", which will prevent actual key after it being used.